src/Common/Auth/AuthHash.php

```diff
@@ -192,9 +192,26 @@ public static function passwordVerify(&$password, $hash)
         if (!empty(preg_match('/^\$6\$rounds=/', $hash))) {
             // Process SHA512HASH algo separately, since uses crypt
             $valid = hash_equals($hash, crypt($password, $hash));
+        } else if (preg_match('/^\$2a\$05\$/', $hash)) {
```

Comment:
hi @dcvidigal ,
Just drop this else if block and then things will be good. Gonna then add some phpunit testing for this function to ensure we don't lose this functionality (support of legacy hashes) in the future :)

Reply:
Ok I think it's all good for unit testing :)